generators/firechip/src/main/scala/Targets.scala

```diff
@@ -0,0 +1,176 @@
+package firesim.firesim
```

Comment:
This file also seems to be executable. `-rwxrwxr-x`. Can you confirm if this is the case on your end? And if so, can we switch it to something like `-rw-rw-r--`

Reply:
I will fix this when i get back to my machine.